okapi-core/src/main/java/org/folio/okapi/MainVerticle.java

```diff
@@ -83,7 +83,8 @@ public void init(Vertx vertx, Context context) {
     super.init(vertx, context);
 
     JsonObject config = context.config();
-    port = Integer.parseInt(Config.getSysConf("port", "9130", config));
+    String portString = System.getProperty("http.port", "9130");
+    port = Integer.parseInt(Config.getSysConf("port", portString, config));
     String okapiVersion2 = Config.getSysConf("okapiVersion", null, config);
```

Comment:
So port is only supported as property. Not as Json configuration. Is that intended?

Reply:
No, good catch!
Fixed.